graphpype/tests/test_utils_dtype_coord.py

```diff
@@ -0,0 +1,38 @@
+from graphpype.utils_dtype_coord import (find_index_in_coords, where_in_coords)
+
+import numpy as np
+
+nb_subj_nodes = 5
+
+subj_coords = np.random.randint(low=-70, high=70, size=(nb_subj_nodes, 3))
+
+corres_coords = np.concatenate(
+    (np.random.randint(low=-70, high=70, size=(5, 3)), subj_coords), axis=0)
+
+nb_corres_nodes = corres_coords.shape[0]
+np.random.shuffle(corres_coords)
+
+
+def test_where_in_coords():
+    """test_where_in_coords"""
+    where_in_corres = where_in_coords(subj_coords, corres_coords)
+
+    assert len(where_in_corres) == nb_subj_nodes, \
+        ("Error, where_in_corres {} should the same as nb_subj_nodes \
```

Comment:
It's very non-standard to have these messages in tests. Tests should be compact and self-explanatory.

Reply:
all the tests I have done so far have messages, I do not why this is an issue for this line in particular

Reply:
not this line in particular. It should probably be changed everywhere